stable/xray/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # JFrog Xray Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [2.3.0] - Jul 13, 2020
```

Comment:
Why does this justify a minor version bump? I think this should be 2.2.2

Reply:
Done